### PR TITLE
Add a place confirmation email for SR2025 teams

### PR DESCRIPTION
**Pull request by PeterJCLaw**
## Summary

This is aimed at teams we're confirming before we have things like Discord set up.

## Checklist

- [x] Has front-matter (`to` and `subject`)
- [x] Compared to a similar email from the previous year
  - Lots of this content was copied from there, plus content from https://github.com/srobo/team-emails/pull/188
- [x] Ensured there's enough context for a rookie team to understand

### CODE REVIEW

File: SR2025/2024-09-04-confirm-early-places.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+watch the presentations, get to know each other and start working together on
+the kit.
+
+As usual we will loan your team a [kit][kit] of parts to help build your robot.
+This year we hope to send out kits shortly before Kickstart, in early October.
+
+In order to enable us to send out your kit you need to agree to our _kit
+disclaimer_ (PDF attached). This disclaimer covers the terms and conditions
+under which you may use the kit and outlines the limits of our responsibilities.
+Please **sign it, scan it and email it back to us**.
```
Comment (PeterJCLaw) [inline SR2025/2024-09-04-confirm-early-places.md:29]: ~boil it, mash it, stick it in a stew~

File: SR2025/2024-09-04-confirm-early-places.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+  - Kit Disclaimer (PDF): https://drive.google.com/file/d/1dZAawbzQwh0peaZRpesL9Up5CzOr--Ec/view
+---
+
+Hi,
+
+You're in! We're happy to confirm you've got a place at Student Robotics 2025!
+
+Thank you for taking on the role of a [team supervisor][team-supervisor] for
+your team. If you have any questions about the [competition programme][programme-structure],
+building a robot or anything else, please feel free to email us at
```
Comment (sedders123) [inline SR2025/2024-09-04-confirm-early-places.md:14]:
```suggestion
building a robot or anything else, please feel free to reach out to us on [Discord][discord] or email us at
```

**Review by sedders123 — DISMISSED** [review]

File: SR2025/2024-09-04-confirm-early-places.md (hunk 2 above)
Comment (PeterJCLaw) [reply SR2025/2024-09-04-confirm-early-places.md]: 62d2956e9016528559075b11b31d4b682ae2d62e

File: SR2025/2024-09-04-confirm-early-places.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+watch the presentations, get to know each other and start working together on
+the kit.
+
+As usual we will loan your team a [kit][kit] of parts to help build your robot.
+This year we hope to send out kits shortly before Kickstart, in early October.
+
+In order to enable us to send out your kit you need to agree to our _kit
+disclaimer_ (PDF attached). This disclaimer covers the terms and conditions
+under which you may use the kit and outlines the limits of our responsibilities.
+Please **sign it, scan it and email it back to us**.
```
Comment (samjmartin04) [inline SR2025/2024-09-04-confirm-early-places.md:30]:
```suggestion
Please **sign it, scan it, and email it back to us**.
```

File: SR2025/2024-09-04-confirm-early-places.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+    SHIPPING_ADDRESS
+
+If your team would like to get started before Kickstart, **new for this year**
+we have prepared some [pre-Kickstart activities][pre-kickstart-activities] which
+introduce robotics and our simulator without needing the kit.
+
+At various points in the year we will have a number of "[Tech Days][tech-days]"
+at locations around the UK. These provide an opportunity for your team to get
+hands-on help from our volunteer mentors, make progress on their robot, and meet
+other teams.
```
Comment (samjmartin04) [inline SR2025/2024-09-04-confirm-early-places.md:44]: Maybe add something saying we'll confirm/inform when and where these are in the coming weeks/months?

File: SR2025/2024-09-04-confirm-early-places.md (hunk 3 above)
Comment (PeterJCLaw) [reply SR2025/2024-09-04-confirm-early-places.md:30]: Thanks for the suggestion, however I'm going to stick with the grammar style as it is.

File: SR2025/2024-09-04-confirm-early-places.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+
+In order to enable us to send out your kit you need to agree to our _kit
+disclaimer_ (PDF attached). This disclaimer covers the terms and conditions
+under which you may use the kit and outlines the limits of our responsibilities.
+Please **sign it, scan it and email it back to us**.
+
+We will be sending your kit to the shipping address you provided when you signed
+up:
+
+    SHIPPING_ADDRESS
```
Comment (PeterJCLaw) [inline SR2025/2024-09-04-confirm-early-places.md:35]: We should also mention that we'll tell them when we send it and that will include a tracking link.

File: SR2025/2024-09-04-confirm-early-places.md [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+<teams@studentrobotics.org>.
+
+This year we're hosting [Kickstart][kickstart] at the University of Southampton,
+on Saturday 12th October. We will also be live-streaming the event if you are
+unable to attend in person. In that case we encourage that your team meet up to
+watch the presentations, get to know each other and start working together on
+the kit.
+
+As usual we will loan your team a [kit][kit] of parts to help build your robot.
+This year we hope to send out kits shortly before Kickstart, in early October.
```
Comment (samjmartin04) [inline SR2025/2024-09-04-confirm-early-places.md:25]:
```suggestion
This year we hope to send out kits shortly before Kickstart, in early October. We will email you when we've dispatched your kit.
```

Comment (PeterJCLaw) [reply SR2025/2024-09-04-confirm-early-places.md]: ff46aec

File: SR2025/2024-09-04-confirm-early-places.md (hunk 5 above)
Comment (PeterJCLaw) [reply SR2025/2024-09-04-confirm-early-places.md]: ff46aec

File: SR2025/2024-09-04-confirm-early-places.md (hunk 4 above)
Comment (PeterJCLaw) [reply SR2025/2024-09-04-confirm-early-places.md]: included in b90a1c7e8ad9c94dc1feb4f646dc3dfd3ba4c1fd

**Comment by PeterJCLaw** [comment]
This now includes the Discord details; copied from last year, so would be good to get them checked in case anything has changed.

File: SR2025/2024-09-11-confirm-places.md [hunk 7]
```diff
@@ -0,0 +1,102 @@
...
+with other teams we have a [Discord][discord] server. This is the primary
+communication method for competitor support, enabling competitors to get support
+directly from our volunteer mentors at any time.
+
+Your invite link is <https://discord.com/invite/TODO_REPLACE_DISCORD_LINK_BEFORE_SENDING>.
+If you don't already have a Discord account, you can register an account for
+free. Accounts should be per individual and not shared with others.
+
+Within Discord there are a number of shared channels, which everyone has access
+to, as well as a channel which is private to your team. Your team's channel is
```
Comment (samjmartin04) [inline SR2025/2024-09-11-confirm-places.md:54]:
```suggestion
to, as well as a channel which is just for your team, which can only be accessed by your team members and our Blueshirt volunteers. Your team's channel is
```

Comment (WillB97) [reply SR2025/2024-09-11-confirm-places.md:54]:
```suggestion
to. Additionally there is a channel which is just for your team, which can only be accessed by your team members and our Blueshirt volunteers. Your team's channel is
```

File: SR2025/2024-09-11-confirm-places.md [hunk 8]
```diff
@@ -0,0 +1,102 @@
...
+called `team-TODO_REPLACE_TLA_BEFORE_SENDING`.
```
Comment (PeterJCLaw) [inline SR2025/2024-09-11-confirm-places.md:55]:
```suggestion
Within Discord there are a number of shared channels, to which everyone has
access. You also have a channel which is accessible only by you, your team and
our Blueshirt volunteers. Your team's channel is called
`team-TODO_REPLACE_TLA_BEFORE_SENDING`.
```

**Pull request merged**